pycm/pycm_obj.py

```diff
@@ -467,7 +473,8 @@ def save_obj(
                          "Matrix": matrix_items,
                          "Digit": self.digit,
                          "Sample-Weight": weights_vector_temp,
-                         "Transpose": self.transpose}
+                         "Transpose": self.transpose,
+                         "Is_imbalanced": self.imbalance}
```

Comment:
`Is_imbalanced` sounds a little weird for me, I suggest `Imbalanced` (Only for output file)
Other parts are okay.

Reply:
OK that's cool.